mitodl/mitxonline
# Delete linked instructor page from product pages

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/4980

### Description (What does it do?)
When an instructor page is deleted, any product page where the deleted instructor page is linked should have the link to the  instructor page removed.

### How can this be tested?

1. Create an instructor page and publish it.
2. Link the instructor page to the product page via the faculty section of the product page.
3. Publish the product page.
4. Delete the instructor page.
5. Visit http://mitxonline.odl.local:8013/api/v2/courses/ and ensure no errors are thrown.
